modules/preview-react/form-field/lib/FormField.tsx

```diff
@@ -18,7 +18,8 @@ export interface FormFieldProps extends Omit<ExtractProps<typeof Stack, never>,
    */
   orientation: 'vertical' | 'horizontal';
   /**
-   * Spacing between children elements
+   * Spacing between children elementsSpacing between children element. The default value is
```

Comment:
Something got messed up addressing feedback here. Run Prettier after adjusting this.
```suggestion
   * Spacing between children element. The default value is
```

Reply:
I think the "Apply Suggestions" in the VSCode Pull Request plugin duplicate in strange ways. I've been manually applying them instead.